pom.xml

```diff
@@ -99,7 +100,7 @@
         <dependency>
             <groupId>io.gatling</groupId>
             <artifactId>gatling-enterprise-plugin-commons</artifactId>
-            <version>0.0.3</version>
+            <version>0.0.3-M20211216111418</version>
```

Comment:
This will never make it on maven central. Sonatype has lost it, period.
You'd better use a SNAPSHOT and push to a dev, not main.

Reply:
Please upgrade to 0.0.3-M20211217085936